class.jetpack-cli.php

```diff
@@ -159,19 +294,290 @@ public function module( $args, $assoc_args ) {
+						jetpack_protect_save_whitelist( $whitelist );
+						WP_CLI::success( __( 'Cleared all whitelisted IPs', 'jetpack' ) );
+					} else {
+						WP_CLI::line( __( 'Whitelist is empty.', "jetpack" ) ) ;
+					}
+					break;
+				}
+
+				// Append new IP to whitelist array
+				array_push( $whitelist, $new_ip );
+
+				// Save whitelist if there are no errors
+				$result = jetpack_protect_save_whitelist( $whitelist );
+				if ( is_wp_error( $result ) ) {
+					WP_CLI::error( __( $result, 'jetpack' ) );
+				}
+
+				WP_CLI::success( sprintf( __( '%s has been whitelisted.', 'jetpack' ), $new_ip ) );
+				break;
+			case 'prompt':
+				WP_CLI::error(
+					__( "No command found.
+						\nPlease enter the IP address you want to whitelist.\nYou can save a range of IPs {low_range}-{high_range}. No spaces allowed.  (example: 1.1.1.1-2.2.2.2)
+						\nYou can also 'list' or 'clear' the whitelist.",
+						'jetpack'
+					)
+				);
+				break;
+		}
+	}
+
+	/**
+	 * Manage Jetpack Options
+	 *
+	 * ## OPTIONS
+	 *
+	 * list   : List all jetpack options and their values
+	 * delete : Delete an option
+	 *          - can only delete options that are white listed.
+	 * update : update an option
+	 *          - can only update option strings
+	 * get    : get the value of an option
+	 *
+	 * ## EXAMPLES
+	 *
+	 * wp jetpack options list
+	 * wp jetpack options get    <option_name>
+	 * wp jetpack options delete <option_name>
+	 * wp jetpack options update <option_name> [<option_value>]
+	 *
+	 * @synopsis <list|get|delete|update> [<option_name>] [<option_value>]
+	 */
+	public function options( $args, $assoc_args ) {
+		$action = isset( $args[0] ) ? $args[0] : 'list';
+		$safe_to_modify = Jetpack::get_jetapck_options_for_reset();
+
+		// Jumpstart is special
+		array_push( $safe_to_modify, 'jumpstart' );
+
+		if ( ! in_array( $action, array( 'list', 'get', 'delete', 'update' ) ) ) {
+			WP_CLI::error( sprintf( __( '%s is not a valid command.', 'jetpack' ), $action ) );
+		}
+
+		if ( isset( $args[0] ) ) {
+			if ( 'get' == $args[0] && isset( $args[1] ) ) {
+				$action = 'get';
+			} else if ( 'delete' == $args[0] && isset( $args[1] ) ) {
+				$action = 'delete';
+			} else if ( 'update' == $args[0] && isset( $args[1] ) ) {
+				$action = 'update';
+			} else {
+				$action = 'list';
+			}
+		}
+
+		// Bail if the option isn't found
+		$option = isset( $args[1] ) ? Jetpack_Options::get_option( $args[1] ) : false;
+		if ( isset( $args[1] ) && ! $option && 'update' !== $args[0] ) {
+			WP_CLI::error( __( 'Option not found or is empty.  Use "list" to list option names', 'jetpack' ) );
+		}
+
+		// Let's print_r the option if it's an array
+		// Used in the 'get' and 'list' actions
+		$option = is_array( $option ) ? print_r( $option ) : $option;
+
+		switch ( $action ) {
+			case 'get':
+				WP_CLI::line( "\t" . str_pad( __( 'Option', 'jetpack' ), 20 ) . __( 'Value', 'jetpack' ) );
+				WP_CLI::line( "\t" . str_pad( $args[1], 20 ) . $option );
```

Comment:
when getting an option, you only need to output the value, rather than building your mock table